Sources/BraintreeSEPADirectDebit/BTSEPADirectDebitTestHelper.swift

```diff
@@ -0,0 +1,66 @@
+import Foundation
+
+///Provides helper functions to test the SEPA Direct Debit flow in sandbox.
+@objcMembers public class BTSEPADirectDebitTestHelper: NSObject {
+    
+    /// Generates a valid 27-digit IBAN (International Bank Account Number) for testing SEPA Direct Debit flows.
+    /// - Returns: a valid IBAN
+    public static func generateValidIBAN() -> String {
+        let countryCode: String = "FR"
```

Comment:
Nitpick - would it be worth exposing the countryCode (maybe providing a default value) for merchants to set using this function? Not sure if there are implications where they support SEPA in some countries but not others.

Reply:
I thought about that but all of the countries have different IBAN lengths/requirements/checks so we would need to build out logic for all of them: https://www.iban.com/structure.